### ISSUE TITLE
[Idea] When will there be a knockback many on the servers will be banned because of this?

### ISSUE THREAD

**Issue by DARKSRG** [MCCTeam/Minecraft-Console-Client#2350]
### Prerequisites

- [X] I have read and understood the [user manual](https://mccteam.github.io/guide/)
- [X] I made sure I am running the latest [development build](https://github.com/MCCTeam/Minecraft-Console-Client/releases/latest)
- [X] I tried to [look for similar feature requests](https://github.com/MCCTeam/Minecraft-Console-Client/issues?q=is%3Aissue) before opening a new one

### Console Client Version

Latest

### Describe your problem

A lot of time has passed since 2017, but you promised to make a knockback on bots. Can people help you with this?

### Minecraft Version

1 19 2

**Comment by milutinke** [comment]
There would have to be a physics systems implemented for knockback to work.
First the whole of protocol library should be refactored, then the movement code and physics implemented.
TLDR, it's not going to be added any time soon.

**Comment by milutinke** [comment]
Also, duplicate of #356
